File: tests/C/0010.c
int main (int argc, char *argv[])
{
	#define N 10
	#define M 20
	#define O 30

	int x[N], Y[M], z[O];

	#pragma acc data present(x)
	{
	}

	#pragma acc data present(Y)
	{
	}

	#pragma acc data copy(x) \
	  copyin(Y) copyout(z)
	{
	}

	#pragma acc \
	data \
present(z)
	{
	}

	#pragma acc data \
  copy(x) copyin (Y) copyout (z)
	{
	}

	return 0;
}

File: tests/Fortran/Fixed/0010.f
      program test

      integer, parameter :: N = 10
      integer, parameter :: M = 20
      integer, parameter :: O = 30
      integer x(N),y(M),z(O)

!$acc data present(x)

!$acc end data

!$acc DATA
!$acc1 PRESENT(y)

!$acc end data

!$acc data
!$acc2 copy(X)
!$acc3 copyin(Y)
!$acc4 copyout(Z)

!$acc end data

!$acc data present (x)
!$acc end data

!$acc data
!$acc& copy (x)
!$acc& copyin(y) 
!$acc& copyout (z)
!$acc end DATA

      end program
